modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -546,6 +550,7 @@ const Main = ({
 	fixedOptions = [],
 	label,
 	localizedValue = {},
+	localizedValueEdited,
```

Comment:
We use this property to know if the field has been modified by the user. For example: If the select field has a predefined value, and the user tries to change back to `'choose an option'`, the `(value?.length === 0 && localizedValueEdited)` will check if the user changed this value and keep the `'choose an option'` option. Otherwise, it would force the value back to the predefinedValue, which is an inconvenient to the user.